themes/jb/layouts/partials/episode/downloadlinks.html

```diff
@@ -0,0 +1,19 @@
+<b>Direct Download: </b><a href="{{.Params.podcast_file}}">MP3</a>
+{{ if (not (eq .Params.podcast_alt_file nil)) }}
+| <a href="{{.Params.podcast_alt_file}}">MP3 mirror</a>
+{{ end }}
+{{ if (not (eq  .Params.podcast_ogg_file nil)) }}
+| <a href="{{.Params.podcast_ogg_file}}">OGG</a>
+{{ end }}
+{{ if (not (eq .Params.video_file nil)) }}
+| <a href="{{.Params.video_file}}">Video</a>
+{{ end }}
+{{ if (not (eq .Params.video_hd_file nil)) }}
+| <a href="{{.Params.video_hd_file}}">HD Video</a>
+{{ end }}
+{{ if (not (eq .Params.video_mobile_file nil)) }}
+| <a href="{{.Params.vide_mobile_file}}">Mobile Video</a>
```

Comment:
Looks like there was a slight spelling error here: the line before this you had `.Params.video_mobile_file` and this line you put `vide` instead.

@aaronrea would you mind doing another small PR to correct?

Reply:
@elreydetoda good catch!

Reply:
good catch and done. #188  thanks!